ui/apps/platform/src/Containers/Vulnerabilities/NodeCves/NodeCve/NodeCvePage.tsx

```diff
@@ -14,19 +15,47 @@ function NodeCvePage() {
     // eslint-disable-next-line @typescript-eslint/no-unused-vars
     const { cveId } = useParams() as { cveId: string };
 
-    const nodeCveName = cveId; // TODO Replace me with queried data
+    const [nodeCveMetadata, setNodeCveMetadata] = useState<NodeCveMetadata>();
+    const nodeCveName = nodeCveMetadata?.cve;
+
+    // TODO - Simulate a loading state, will replace metadata with results from a query
+    useEffect(() => {
+        setTimeout(() => {
+            setNodeCveMetadata({
+                cve: cveId,
+                firstDiscoveredInSystem: '2021-01-01T00:00:00Z',
+                distroTuples: [
+                    {
+                        summary: 'This is a sample description used during development',
+                        link: `https://access.redhat.com/security/cve/${cveId}`,
+                        operatingSystem: 'rhel',
+                    },
+                ],
+            });
+        }, 1500);
+    }, []);
 
     return (
         <>
             <PageTitle title={`Node CVEs - NodeCVE ${nodeCveName}`} />
             <PageSection variant="light" className="pf-u-py-md">
                 <Breadcrumb>
                     <BreadcrumbItemLink to={workloadCveOverviewCvePath}>CVEs</BreadcrumbItemLink>
-                    <BreadcrumbItem isActive>{nodeCveName} </BreadcrumbItem>
+                    <BreadcrumbItem isActive>
+                        {nodeCveName ?? (
+                            <Skeleton screenreaderText="Loading CVE name" width="200px" />
+                        )}
+                    </BreadcrumbItem>
                 </Breadcrumb>
             </PageSection>
             <Divider component="div" />
-            <PageSection variant="light"></PageSection>
+            <PageSection variant="light">
+                <NodeCvePageHeader data={nodeCveMetadata} />
```

Comment:
Can this be reused for Platform CVEs too?

Reply:
Ah, potentially. It looks like the structure is the same in Workload CVEs too. I'll see if I can refactor this into a common CVEPageHeader in a follow up.